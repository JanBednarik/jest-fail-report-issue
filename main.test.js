beforeEach(() => {
    throw new Error('Fail!');
});

test('1 is 1', () => {
    expect(1).toBe(1);
});

test('1 is 2', () => {
    expect(1).toBe(2);
});

test('1 is 3', () => {
    expect(1).toMatchSnapshot();
});
