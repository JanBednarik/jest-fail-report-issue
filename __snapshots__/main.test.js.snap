// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1 is 3 1`] = `3`;
